src/server.mjs

```diff
@@ -46,6 +46,7 @@ function getReqBody(req) {
 
 async function proxy(host, request, reqBody, response) {
   let pathname = url.parse(request.url).pathname;
+  pathname = pathname === "/" ? "" : pathname; // to avoid a trailing slash
```

Comment:
I just wonder what we actually need a path at all for?

Reply:
okay, yeah, this might be right, but we can deal with in the future.